TOOLS_LICENSES.md

```diff
@@ -0,0 +1,47 @@
+# Tools licenses
```

Comment:
We don't have a file listing software used by our tools but not included in games built using Defold. Since I'm now adding ANTLR I thought we might as well add such a file.